CHANGELOG.md

```diff
@@ -4,11 +4,12 @@
 
 ### Breaking Changes
 - Add a new CLI option to limit the number of requests in a single RPC batch request. Default=1 [#4965](https://github.com/hyperledger/besu/pull/4965)
-- Changed JsonRpc http service to return the error -32602 (Invalid params) with a 200 http status code
+- Change JsonRpc http service to return the error -32602 (Invalid params) with a 200 http status code
 - Besu requires minimum Java 17 and up to build and run [#3320](https://github.com/hyperledger/besu/issues/3320)
 - PKCS11 with nss module (PKCS11 based HSM can be used in DevP2P TLS and QBFT PKI) does not work with RSA keys 
 in Java 17. SoftHSM is tested manually and working. (Other PKCS11 HSM are not tested). The relevant unit and acceptance 
 tests are updated to use EC private keys instead of RSA keys.
+- Change eth_feeHistory parameter `blockCount` to accept hexadecimal string #5047
```

Comment:
```suggestion
- Change eth_feeHistory parameter `blockCount` to accept hexadecimal string [#5047](https://github.com/hyperledger/besu/issues/5047)
```

Reply:
I made the above suggestion - but I think your way also works because we're in GitHub and it's much simpler...

Reply:
maybe say why it's a breaking change - accept hexadecimal string (was accepting plain integer)

Reply:
Done, ty